src/engine/renderer/tr_scene.cpp

```diff
@@ -493,6 +493,129 @@ void RE_RenderScene( const refdef_t *fd )
 		Sys::Drop( "R_RenderScene: NULL worldmodel" );
 	}
 
+	refdef_t refdef{};
```

Comment:
This debugging code should go in its own function

Reply:
Done now.